packages/react-core/src/components/index.ts

```diff
@@ -51,7 +51,7 @@ export * from './MultipleFileUpload';
 export * from './Nav';
 export * from './NotificationBadge';
 export * from './NotificationDrawer';
-export * from './OptionsMenu';
+export * from '../deprecated/components/OptionsMenu';
 export * from './OverflowMenu';
```

Comment:
@nicolethoen @tlabaj - this is one of the things i was questioning the most. should we keep this so we can still import OptionsMenu the same way?

Reply:
I think it should be added to the index.ts in the deprecated directory. it's set up so that next and deprecated components get built into their own directories alongside the components. So we don't export them from the component directory's index.ts